cogs/fun.py

```diff
@@ -224,7 +292,7 @@ async def yesno(self, ctx: Context, *, question: str) -> None:
                     await ctx.send(f"OMFG! [STATUS : {meme.status}]")
 
     @command(aliases=["dadjoke", "awdad", "dadpls", "shitjoke", "badjoke"])
-    async def joke(self, ctx: Context) -> None:
+    async def dadjoke(self, ctx: Context) -> None:
```

Comment:
Since command name is changed to `dadjoke` there is no point in keeping `"dadjoke" ` in aliases